src/lib/viewers/box3d/Box3D.scss

```diff
@@ -1,3 +1,12 @@
+.bp-is-mobile {
+    .bp-box3d .bp-controls {
+        /* stylelint-disable declaration-no-important */
+        opacity: 1 !important;
+        /* stylelint-enable declaration-no-important */
+        z-index: 1;
```

Comment:
@JustinHoldstock any issues with using z-index: 1? Moving this block above the vr-enabled override fixes the first issue.